plugins/bbcode/plugin.js

```diff
@@ -660,11 +656,7 @@
 								tagName = 'color';
 								value = CKEDITOR.tools.convertRgbToHex( value );
 							} else if ( ( value = style[ 'font-size' ] ) ) {
-								var percentValue = value.match( /(\d+)%$/ );
```

Comment:
this looks for a digit with a percent and then sets the value and tag name. Instead we just set the tag name and trust the values we setup for our editor.